docs/biovalidator_benchmarks/2023.01.23_benchmarks/EGAs_endpoint/README.md

```diff
@@ -0,0 +1,68 @@
+# JSON validation summary
+The script [``create_benchmarks.py``](https://github.com/EbiEga/ega-metadata-schema/blob/main/.github/scripts/create_benchmarks.py) validated all JSON files in a directory against a validation server endpoint, recording valuable parameters of its execution.
+Overall execution parameters:
+| Date                | Passed validation   |   Validation time (s) | Input directory                  | Used endpoint                              |   Nº of executions |   Nº of files |   Nº of properties |   Nº of ontology validations |
+|:--------------------|:--------------------|----------------------:|:---------------------------------|:-------------------------------------------|-------------------:|--------------:|-------------------:|-----------------------------:|
+| 2023-01-23 19:04:19 | True                |                2948.2 | 2023.01.23_benchmarks/json_docs/ | http://biovalidator.ega.ebi.ac.uk/validate |                250 |            48 |           15317076 |                       695043 |
+
+All files generated in this execution are stored in this directory (``2023.01.23_benchmarks/EGAs_endpoint``) with this same README. These files are:
+1. Two CSV files: the complete set of validation parameters for each file and validation attempt ([``2023.01.23_benchmarks/EGAs_endpoint/complete_df.csv``](summary_df.csv)) and a summary of those ([``2023.01.23_benchmarks/EGAs_endpoint/summary_df.csv``](summary_df.csv)), also displayed below.
+2. Three graphs (all inserted below) of the full set of parameters: a 3D graph ([``2023.01.23_benchmarks/EGAs_endpoint/3D_scatterPlot.png``](3D_scatterPlot.png)) of the validation time, number of properties, and number of ontology validations; two 2D scatter subplots ([``2023.01.23_benchmarks/EGAs_endpoint/2D_scatterPlot.png``](2D_scatterPlot.png)) of the validation time versus the other two variables; and a 2D density plot ([``2023.01.23_benchmarks/EGAs_endpoint/2D_densityPlots.png``](2D_densityPlots.png)) containing 2 subplots: one for each variable versus the validation time
+
+Graphical representations:
+
+![3D Scatter plot](3D_scatterPlot.png)
+![2D Scatter plots](2D_scatterPlot.png)
+![2D Density plot](2D_densityPlots.png)
```

Comment:
As compared to the local BioV endpoint, using the EGA service looks to have more variation and higher chance of longer validation times. Although, it appears to still scale linearly (which is good).

Reply:
Yep, same diagnosis as in the ticket's comment. More outliers and a higher validation time. This may be significant and worthy of each big node/submitter to have their own local endpoint, given how easy it is to deploy.